src/main/java/seedu/address/logic/commands/KeyCommand.java

```diff
@@ -0,0 +1,71 @@
+package seedu.address.logic.commands;
+
+//@@author lohtianwei
+import java.nio.charset.StandardCharsets;
+import java.util.logging.Logger;
+
+import com.google.common.hash.Hashing;
+
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.UserPrefs;
+
+/**
+ * Sets lock in model to true/false depending on current state
+ */
+public class KeyCommand extends Command {
+    public static final String COMMAND_WORD = "key";
+    public static final String COMMAND_ALIAS = "k";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Toggles the lock on MTM. "
+            + "Functionality of MTM reduced\n"
+            + "Input empty password to see current lock state\n"
```

Comment:
I think just stating that by entering the command itself, will indicate the status of the lock.